### DISCUSSION TITLE
Revoke database credentials

### DISCUSSION THREAD

**Discussion by maheshj01** [supabase/supabase#2992] [Questions]
How to revoke and regenerate the existing database credentials mainly

JWTSecret, String API key,ServiceRole

Thanks

**Comment by dshukertjr** [comment]
Hi @maheshmnj!

If I am not wrong, you have to contact Supabase team to do this. You can send them an email at `support@supabase.com`.

**Reply by maheshj01** [reply]
Ohh I see thanks for the reply, But It would be nice to integrate it into the supabase dashboard itself.
